app/controllers/api/base_controller/generic.rb

```diff
@@ -224,6 +224,13 @@ def set_ownership_action(klass, type, id, ownership)
       rescue => err
         action_result(false, err.to_s)
       end
+
+      def service_template_workflow(service_template, service_request)
```

Comment:
Since this is used by a couple of different controllers now, moving this into generic for now.